.github/workflows/test.yml

```diff
@@ -50,5 +49,8 @@ jobs:
     - name: Print out package info to help with debug
       run: pip list
 
-    - name: Run unit tests with pytest
```

Comment:
add windows back in once you figure out how to ignore the paic2 directory in Windows